client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from 'react';
+import { AppBar, Toolbar, Avatar, Button, Box } from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import { useAuth } from '../../context/useAuthContext';
+import logo from '../../Images/logo.png';
+import profile from '../../Images/profile.png';
+import useStyles from './useStyles';
+import { useHistory } from 'react-router-dom';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+const Navbar = (): JSX.Element => {
+  const { logout } = useAuth();
+  const classes = useStyles();
+  const history = useHistory();
+  const { location } = history;
+  const showAuthButtons = location.pathname == '/login' || location.pathname == '/signup' ? true : false;
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <AppBar position="static" className={classes.root}>
+      <Toolbar>
+        <Box className={classes.box}>
+          <img src={logo} alt="logo" className={classes.logo} />
+        </Box>
+
+        {!showAuthButtons && (
+          <Box className={classes.navButtons} justifyContent="space-between" alignItems="center">
+            <Button
+              size="large"
+              onClick={() => {
+                history.push('/discover');
```

Comment:
Try to use React `Link` to route to different pages as opposed to the history object.

Reply:
Yes, I saw that we were using that in Auth header but I dont know the difference between them...Is there any?

Reply:
It's just a minor thing, nothing to worry about. Please see below for a better understanding: 

Ref: https://stackoverflow.com/questions/51116747/react-router-link-vs-redirect-vs-history

Reply:
Thanks for clearing up, instead of refactoring whole thing (leading to more complications). I tried this ``<Button component={Link} to={''} />``. Let me know if I should use this approach.

Reply:
I am going to merge this in now, pls let me know if you think we should change it. Thanks